internal/nexodus/ctlpeers.go

```diff
@@ -0,0 +1,31 @@
+package nexodus
+
+import (
+	"encoding/json"
+	"fmt"
+	"runtime"
+)
+
+func (ac *NexdCtl) ListPeers(_ string, result *string) error {
+	var iface string
+
+	if runtime.GOOS == Darwin.String() {
+		iface = darwinIface
+	} else {
+		iface = wgIface
+	}
+
+	peers, err := DumpPeers(iface)
```

Comment:
This won't work for `nexd proxy`. I can think of a few options for this PR. I am OK with any of them. If you choose #1 or #2, please file an issue so I can come back and implement it later.

1. Check `ac.ax.userspaceMode` and return an error.

2. Add a fallback implementation if `userspaceMode` is true that returns data we have in our peer cache. We'd have the IPs and such, but not state like last handshake time.

3. Use the userspace stack configuration interface to get the data you want. For reference, see https://www.wireguard.com/xplatform/#configuration-protocol and search for `IpcGet()`. If you use `IpcGet()` you will have all the data needed.

4. It's possible that the wgctl library would work, too. The README says it can speak the userspace configuration protocol, so I'm wondering if we can get it to work.

Reply:
Good catch. I am choosing the easy path and opened an issue one of us can circle back on. Mainly because I need the functionality we removed with the wg binary for tshooting CI output in QA this week. Ty.

Reply:
Yeah, that's good with me! I figured you would. I just needed to get the options / ideas for how to do it out of my head.